instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/instrumenter/http/HttpServerMetrics.java

```diff
@@ -51,19 +48,19 @@ public static RequestMetrics get() {
   }
 
   private final LongUpDownCounter activeRequests;
-  private final DoubleValueRecorder duration;
+  private final DoubleHistogram duration;
```

Comment:
Is histogram always the one to pick instead of value recorder?

Reply:
yes, ValueRecorder has been renamed to Histogram. Semantics should be identical (although maybe no negative recordings allowed on the Histogram?).

Reply:
Currently histogram shouldn't have negative recordings.